flows/engine/assets.go

```diff
@@ -0,0 +1,66 @@
+package engine
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/flows/definition"
+)
+
+type assets struct {
```

Comment:
I am warming to this name, though fear that it might be confusing to others due to assets commonly being used for graphics / css etc.. Not sure I have a better suggestion though so that doesn't make a very powerful argument.